callbacks/control_bar.py

```diff
@@ -97,3 +97,35 @@ def annotation_visibility(checked, store, figure, image_idx):
         patched_figure["layout"]["shapes"] = []
 
     return store, patched_figure
+
+
+clientside_callback(
+    """
+    function dash_filters_clientside(brightness, contrast) {
+    console.log(brightness, contrast)
+        js_path = "#image-viewer > div.js-plotly-plot > div > div > svg:nth-child(1) > g.cartesianlayer > g > g.plot > g"
+        changeFilters(js_path, brightness, contrast)
+        return ""
+    }
+    """,
+    Output("dummy-output", "children", allow_duplicate=True),
+    Input("figure-contrast", "value"),
+    Input("figure-brightness", "value"),
+    prevent_initial_call=True,
+)
+
+
+@callback(
+    Output("figure-brightness", "value", allow_duplicate=True),
+    Output("figure-contrast", "value", allow_duplicate=True),
+    Output("colormap-scale", "value", allow_duplicate=True),
```

Comment:
`Output("colormap-scale", "value")` will be removed in the following PR as per the project requirement (this will then stop this callback from reporting the figure). alternatively, it can also easily be made clientside in the next pr)